doc/source/user/c-info.beyond-basics.rst

```diff
@@ -173,7 +173,7 @@ each array is for PyArray_ITER_NEXT to be called for each of the inputs. This
 incrementing is automatically performed by
 :c:func:`PyArray_MultiIter_NEXT` ( ``obj`` ) macro (which can handle a
 multiterator ``obj`` as either a :c:type:`PyArrayMultiObject *` or a
```

Comment:
Haha! That's funny because there is no `PyArrayMultiObject` defined so any link (with or without `*`) couldn't work. 😄 

However from my understanding `:c:type::`-declared variables can have the `*` and link correctly. See https://docs.scipy.org/doc/numpy/reference/c-api.array.html#data-type-checking for working `PyArrayObject *`-link.